components/teams-service/storage/conformance/conformance_test.go

```diff
@@ -322,16 +320,13 @@ func testAddUsers(ctx context.Context, t *testing.T, s storage.Storage) {
 	_, err := s.StoreTeam(ctx, id, id+" Name", emptyProjectsList)
 	require.NoError(t, err, "setup: failed to create test team")
 
-	users := []string{"one", "two", "three"}
-	team, err := s.AddUsers(ctx, id, users)
-	assert.NotNil(t, team)
+	userIDs := []string{"one", "two", "three"}
+	updatedUserIDs, err := s.AddUsers(ctx, id, userIDs)
+	assert.NotNil(t, updatedUserIDs)
 
-	userIDs, err := s.GetUserIDsForTeam(ctx, id)
-	require.NoError(t, err)
-
-	assert.Equal(t, len(users), len(userIDs), "number of users added does not match")
-	for _, user := range users {
-		assert.Contains(t, userIDs, user, "user %q not added", user)
+	assert.Equal(t, len(userIDs), len(updatedUserIDs), "number of users added does not match")
```

Comment:
do we have a test that adds users, then adds them again to confirm we get the *new* list back?

Reply:
we do have a test that makes sure we don't duplicate user IDs if we add the same user twice? I'm not sure if that's the same case you mean